recipes/ogre/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This file isn't needed as conda-build will automatically get it from the source tarball instead.